### PR TITLE
/static/features TLC

### LINKED ISSUE #7188: 'How it works' page (/static/features) could use some TLC
https://habitica.com/static/features ([code](https://github.com/HabitRPG/habitrpg/blob/develop/website/views/static/features.jade)) has screenshots of the old mobile apps, and the boss image is broken. Having buttons sprinkled throughout a paragraph also doesn't look great IMHO:

image

I know there's currently a code freeze for the new API; I can probably tackle this once APIv3 is out (assuming it won't be for a few weeks/until the end of May).

### PR DESCRIPTION
**Pull request by librarianmage**
Fixes #7188 
### Changes
- [x] Fix grammatical errors / Inconsistencies
- [x] Improve meaning
- [X] ~~Update misc. pictures~~
  - [x] ~~Apps~~
  - [x] ~~Chrome extension~~
  - [x] ~~Website~~
- [x] ~~Fix link paragraph~~
- [x] ~~`TODO`s in page~~
  - [x] ~~Achievements~~
  - [x] ~~Images on bottom part~~
  - [x] ~~Add landing section~~
- [x] ~~Badges?~~
  - [x] [iPhone](https://developer.apple.com/app-store/marketing/guidelines/)
  - [x] [Android](https://play.google.com/intl/en_us/badges/)
  - [x] [Chrome Extensions](https://developer.chrome.com/webstore/branding)
### Photos

---

UUID: 95f16006-1504-4a9f-95b6-0c7314dbd7ee

### CODE REVIEW

File: common/locales/en/front.json [hunk 1]
```diff
@@ -97,24 +97,24 @@
...
-    "marketing3Header": "Apps",
+    "marketing3Header": "Apps and Tools",
     "marketing3Lead1": "The <strong>iPhone & Android</strong> apps let you take care of business on the go. We realize that logging into the website to click buttons can be a drag.",
     "marketing3Lead2": " Other <strong>3rd Party Tools</strong> tie Habitica into various aspects of your life. Our API provides easy integration for things like the <a href='https://chrome.google.com/webstore/detail/habitica/pidkmpibnnnhneohdgjclfdjpijggmjj?hl=en-US' target='_blank'>Chrome Extension</a>, for which you lose points when browsing unproductive websites, and gain points when on productive ones. <a href='http://habitica.wikia.com/wiki/App_and_Extension_Integrations' target='_blank'>See more here</a>",
     "marketing4Header": "Organizational Use",
-    "marketing4Lead1": "Education is one of the best sectors for gamification. We all know how glued to phones and games students are these days; harness that power! Pit your students against eachother in friendly competition. Reward good behavior with rare prizes. Watch their grades and behavior soar.",
```
Comment (crookedneighbor) [inline common/locales/en/front.json]: "sectors" is correct here, I believe.

**Review by crookedneighbor — CHANGES_REQUESTED** [review]

File: common/locales/en/front.json [hunk 2]
```diff
@@ -97,24 +97,24 @@
...
     "marketing1Lead1": "Habitica is a video game to help you improve real life habits. It \"gamifies\" your life by turning all your tasks (habits, dailies, and to-dos) into little monsters you have to conquer. The better you are at this, the more you progress in the game. If you slip up in life, your character starts backsliding in the game.",
-    "marketing1Lead2": "<strong>Get Sweet Gear</strong>. Improve your habits to build up your avatar. Show off the sweet gear you've earned",
+    "marketing1Lead2": "<strong>Get Sweet Gear</strong>. Improve your habits to build up your avatar. Show off the sweet gear you've earned!",
     "marketing1Lead2Title": "Get Sweet Gear",
-    "marketing1Lead3": "<strong>Find Random Prizes</strong>. For some, it's the gamble which motivates them, a system called \"stochastic rewarding\". Habitica accommodates all reinforcement styles: positive, negative, predictable, and random.",
+    "marketing1Lead3": "<strong>Find Random Prizes</strong>. For some, it's the gamble which motivates them: a system called \"stochastic rewards\". Habitica accommodates all reinforcement and punishment styles: positive, negative, predictable, and random.",
```
Comment (khipkin) [inline common/locales/en/front.json]: Not sure about the addition of "and punishment" here.

File: common/locales/en/front.json (hunk 1 above)
Comment (librarianmage) [reply common/locales/en/front.json]: Fixed

File: common/locales/en/front.json [hunk 3]
```diff
@@ -97,24 +97,24 @@
...
     "marketing4Header": "Organizational Use",
-    "marketing4Lead1": "Education is one of the best sectors for gamification. We all know how glued to phones and games students are these days; harness that power! Pit your students against eachother in friendly competition. Reward good behavior with rare prizes. Watch their grades and behavior soar.",
+    "marketing4Lead1": "Education is one of the best sectors for gamification. We all know how glued to phones and games students are these days; harness that power! Pit your students against each other in friendly competition. Reward good behavior with rare prizes. Watch their grades and behavior soar.",
     "marketing4Lead1Title": "Gamification In Education",
-    "marketing4Lead2": "Health care costs are on the rise, and something's gotta give. Hundreds of programs are built to reduce costs and improve wellness. We believe Habitica can pave a substantial path towards healthy lifestyles.",
+    "marketing4Lead2": "Health care costs are on the rise, and something's gotta give. Hundreds of programs are built to reduce costs and improve wellness; we believe Habitica can pave a substantial path towards healthy lifestyles.",
```
Comment (khipkin) [inline common/locales/en/front.json]: Wasn't really wrong before.  I think it works either way.

Comment (librarianmage) [reply common/locales/en/front.json]: Sorry, I was trying to make it flow better. I know it isn't a grammatical error, I just thought it sounded better.

File: common/locales/en/front.json (hunk 2 above)
Comment (librarianmage) [reply common/locales/en/front.json]: I don't know, but I thought if we are using technical terminology (stochastic rewards), I would change it so that it uses reinforcement/punishment [correctly](https://en.wikipedia.org/wiki/Reinforcement#Terminology). If you think I should change it, I will.

File: common/locales/en/front.json (hunk 3 above)
Comment (khipkin) [reply common/locales/en/front.json]: I understand! :+1: :smile:

File: common/locales/en/front.json (hunk 2 above)
Comment (khipkin) [reply common/locales/en/front.json]: @lemoness @Alys ?

File: common/locales/en/front.json [hunk 4]
```diff
@@ -97,24 +97,24 @@
...
     "marketing1Lead1": "Habitica is a video game to help you improve real life habits. It \"gamifies\" your life by turning all your tasks (habits, dailies, and to-dos) into little monsters you have to conquer. The better you are at this, the more you progress in the game. If you slip up in life, your character starts backsliding in the game.",
-    "marketing1Lead2": "<strong>Get Sweet Gear</strong>. Improve your habits to build up your avatar. Show off the sweet gear you've earned",
+    "marketing1Lead2": "<strong>Get Sweet Gear</strong>. Improve your habits to build up your avatar. Show off the sweet gear you've earned!",
     "marketing1Lead2Title": "Get Sweet Gear",
-    "marketing1Lead3": "<strong>Find Random Prizes</strong>. For some, it's the gamble which motivates them, a system called \"stochastic rewarding\". Habitica accommodates all reinforcement styles: positive, negative, predictable, and random.",
+    "marketing1Lead3": "<strong>Find Random Prizes</strong>. For some, it's the gamble which motivates them: a system called \"stochastic rewards\". Habitica accommodates all reinforcement/punishment styles: positive, negative, predictable, and random.",
```
Comment (mandyzhou) [inline common/locales/en/front.json]: Additional edit: should be "it's the gamble _that_ motivates them".

File: common/locales/en/front.json (hunk 3 above)
Comment (mandyzhou) [inline common/locales/en/front.json]: Hmmm, I like the period better. The period suggests that there's something that distinguishes Habitica from the hundreds of programs mentioned in the previous sentence. This separation is not clear with the semicolon. Additionally, I think short sentences are generally more effective in marketing language.

Comment (mandyzhou) [reply common/locales/en/front.json]: I think it might actually be a good idea to rephrase this to be more clear/informative. It's not clear what those programs are (health care assistance programs? computer software?) and how exactly Habitica can help.

**Comment by crookedneighbor** [comment]
I think the string changes are fine, but I'm not sure we should do much work on the views themselves, especially since we've now actually started work on the new client code.

**Comment by MathWhiz** [comment]
How long do you think it will take?

**Comment by khipkin** [comment]
Suggestion: In `front.json`, line 15: " Put"-->"Put" (note the leading space after the quotes)

**Comment by librarianmage** [comment]
@khipkin I would, but it isn't part of  `/static/features`'s strings

**Comment by librarianmage** [comment]
@crookedneighbor Ok, I will revert all of my changes to `features.jade`

**Comment by librarianmage** [comment]
@crookedneighbor Does that mean that you don't think I should update the pictures?

**Comment by crookedneighbor** [comment]
I don't think we should update the pictures yet, as we don't know yet what the redesigned features page will look like yet. That will influence what should be on the page. 

@Alys @lemoness could one or both of you review the locale changes and make sure they are :+1:

**Review by crookedneighbor — APPROVED** [review]

**Comment by lemoness** [comment]
One small typo:

" a system called \"stochastic rewards\"." should be " a system called \"stochastic rewards."" Other than that, they look good.

**Comment by librarianmage** [comment]
@crookedneighbor @lemoness Updated!

File: website/common/locales/en/front.json [hunk 5]
```diff
@@ -98,22 +98,22 @@
     "loginFacebookAlt": "Sign in with Facebook",
     "loginGoogleAlt": "Sign in with Google",
     "logout": "Log Out",
-    "marketing1Header": "Improve Your Habits By Playing A Game",
```
Comment (crookedneighbor) [inline website/common/locales/en/front.json:101]: @Alys @lemoness what do you think of these wording changes?

Comment (Alys) [reply website/common/locales/en/front.json:101]: I like them.

**Comment by crookedneighbor** [comment]
Notated.
